src/command/graph/check.rs

```diff
@@ -29,6 +32,21 @@ pub struct Check {
     #[structopt(long, short = "s", parse(try_from_str = parse_schema_source))]
     #[serde(skip_serializing)]
     schema: SchemaSource,
+
+    /// Minimum number of requests within the time window for a query to be
+    /// considered
+    #[structopt(long, parse(try_from_str = parse_query_count_threshold))]
+    query_count_threshold: Option<i64>,
+
+    /// Minimum percentage of requests within the time window for a query to be
+    /// considered, relative to total request count. Expected values are 0-5
+    /// (i.e. minimum 5% of total request volume)
+    #[structopt(long, parse(try_from_str = parse_query_percentage_threshold))]
+    query_percentage_threshold: Option<f64>,
+
+    /// Size of the time window with which to validate schema against (in seconds)
+    #[structopt(long, parse(try_from_str = parse_validation_period))]
+    validation_period: Option<ValidationPeriod>,
```

Comment:
Takes a number in seconds, and converts it to a `{ to: -0, from: <seconds> }` along with validating the value

Reply:
i think this would be a lot more ergonomic if you could pass strings like `1h` `10m` `300s` `2d 5h` etc. im sure theres a crate that would help with that (maybe even `chrono`). doesn't _need_ to make it for this release, but i think it'd be a nice touch.

Reply:
I agree, and that was definitely the point of the old cli supporting ISO duration formatting (although I found that hard to understand at first too). I tried finding a way to support those durations in chrono, but couldn't figure it out. I'd be happy to make a second pass at this sometime, since it is a handy ergonomic

Reply:
there's a crate called humantime that might do this for us - no need to get it in this pr though

Reply:
Opened an issue to track this :) #246